coldfront/core/user/forms.py

```diff
@@ -95,25 +95,35 @@ def clean_email(self):
             raise forms.ValidationError(mark_safe(message))
         return email
 
-    def clean_middle_name(self):
-        cleaned_data = super().clean()
-        self.middle_name = cleaned_data.pop('middle_name', '')
```

Comment:
I think the middle name is getting dropped during registration because of the change here. Do you see the same behavior?

The instance's `middle_name` attribute is being set here, and then used in the `save` method, but after the change, this attribute isn't getting set. The user's middle name gets set to the predefined empty string from the `__init__` method as  a result.

By contrast, `self.phone_number` is still being set, and is therefore correctly saved.